google_takeout_parser/parse_json.py

```diff
@@ -186,15 +193,15 @@ def _parse_semantic_location_history(p: Path) -> Iterator[Res[PlaceVisit]]:
     timelineObjects = json_data.get("timelineObjects", [])
     for timelineObject in timelineObjects:
         if "placeVisit" not in timelineObject:
-            # yield RuntimeError(f"PlaceVisit: no 'placeVisit' key in '{p}'")
+            yield RuntimeError(f"PlaceVisit: no 'placeVisit' key in '{p}'")
             continue
         placeVisit = timelineObject["placeVisit"]
         missing_key = _check_required_keys(placeVisit, _sem_required_keys)
         if missing_key is not None:
             yield RuntimeError(f"PlaceVisit: no '{missing_key}' key in '{p}'")
             continue
         try:
-            location_json = placeVisit["location"]
+            location_json = placeVisit.get("location")
```

Comment:
same deal here, just revert to direct access unless you have some data that doesnt have a location. the `_check_required_keys` function expects a dictionary, and if this failed it would pass `None` which would just crash. It crashing on the `KeyError` is easier to debug